Scripts/NuGet/EdFi.Installer.SandboxAdmin/EdFi.Installer.SandboxAdmin.nuspec

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 <package>
   <metadata>
-    <id>EdFi.Installer.SandboxAdmin</id>
+    <id>EdFi.Suite3.Installer.SandboxAdmin</id>
```

Comment:
Can you use file-content-replacer  here https://github.com/rshillington/file-content-replacer  instead of hard coding on this file . 

**/*.nuspec

Find what:  ?<=<id>)(.*?)(EdFi)(?=\b.*</id>)

Replace with %odsapi.package.suitenumber% wih Suite3